solutions/ielfurlan/s015.py

```diff
@@ -0,0 +1,12 @@
+a = 3
+for b in range(1,9):
```

Comment:
Why range 1-9? Write a comment explaining why.
BTW, there is an extensive discussion in the programming community about good comments. As a rule of thumb, good comments do not explain what your code does (your code should be clear enough), but good comments explain why you are using that logic/algorithm etc.
Remember, code is like jokes: if you need to explain, it's not good.

Reply:
The "for in range" loop , the final count is not included and it starts at 0, unless you tell it to start at a different number.